### DISCUSSION TITLE
Broken page  devices\interfaces

### DISCUSSION THREAD

**Discussion by art-leach** [netbox-community/netbox#16836] [Help Wanted!]
Hi All,
We have a broken page with the error below. Has anyone seen this before? 
<class 'AttributeError'>

'NoneType' object has no attribute 'choices'

Python version: 3.8.10
NetBox version: 3.7.5
Plugins: None installed

Best Regards
Arthur Leach

**Comment by candlerb** [comment]
> Has anyone seen this before?

Yes. [Search](https://github.com/netbox-community/netbox/discussions?discussions_q=%27NoneType%27+object+has+no+attribute+%27choices%27) for that error message and you will find #13654
